backend/.env.test

```
@@ -0,0 +1,17 @@
+PORT=3011
+IP=127.0.0.1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IP key should go before the PORT key

(UnorderedKey)